framework/DataObjects/DataSet.py

```diff
@@ -1296,6 +1299,9 @@ def _getRealizationFromCollectorByIndex(self,index):
     assert(self._collector is not None)
     assert(index < len(self._collector))
     rlz = dict(zip(self._orderedVars,self._collector[index]))
+    # don't forget the aligned indices!
```

Comment:
see above

Reply:
Clarified a bit in case a developer hasn't noted how the aligned indices work yet.